ngraph_bridge/ngraph_builder.cc

```diff
@@ -538,9 +541,6 @@ static Status TranslateBinaryOp(
   std::shared_ptr<ng::Node> ng_lhs, ng_rhs;
   TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_lhs, &ng_rhs));
 
-  std::tie(ng_lhs, ng_rhs) =
```

Comment:
So we removed explicit broadcasting here but some ops call TranslateBinaryOp that were not upgraded.  Like Power. Do they need separate handling?

Reply:
We only upgraded ops that were necessary to pass tests at the moment, rest will be done in separate PR's if that's what you are asking.

Reply:
Thinking why the tests for the ops that were not upgraded are not failing. Maybe they don't check broadcasting. This op does seem like it can support broadcast since it is doing element-wise Power
https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/pow.cc 

You can add another function similar to this and call that for the upgraded ops, that way it wont break the ops that are not upgraded.